diff --git a/src/backend.rs b/src/backend.rs
index 663f9c1..ff4c693 100644
--- a/src/backend.rs
+++ b/src/backend.rs
@@ -6,10 +6,53 @@ use std::{env, process};
 pub use models::*;
 mod models;
 
-static PUZZLES: LazyLock<RwLock<PuzzleSolutions>> =
-    LazyLock::new(|| RwLock::new(PuzzleSolutions::new()));
+static PUZZLES: LazyLock<RwLock<PuzzleSolutions>> = LazyLock::new(|| {
+    RwLock::new(PuzzleSolutions::from([
+        (
+            "1".to_string(),
+            Puzzle {
+                solution: "10".to_string(),
+                value: 100,
+            },
+        ),
+        (
+            "2".to_string(),
+            Puzzle {
+                solution: "20".to_string(),
+                value: 200,
+            },
+        ),
+        (
+            "3".to_string(),
+            Puzzle {
+                solution: "30".to_string(),
+                value: 300,
+            },
+        ),
+        (
+            "4".to_string(),
+            Puzzle {
+                solution: "40".to_string(),
+                value: 400,
+            },
+        ),
+    ]))
+});
 
-static TEAMS: LazyLock<RwLock<TeamsState>> = LazyLock::new(|| RwLock::new(TeamsState::new()));
+static TEAMS: LazyLock<RwLock<TeamsState>> = LazyLock::new(|| {
+    RwLock::new(TeamsState::from([
+        ("feco".to_string(), SolvedPuzzles::from(["1".to_string()])),
+        (
+            "jero".to_string(),
+            SolvedPuzzles::from(["1".to_string(), "2".to_string(), "3".to_string()]),
+        ),
+        (
+            "karo".to_string(),
+            SolvedPuzzles::from(["1".to_string(), "3".to_string()]),
+        ),
+        ("genyo".to_string(), SolvedPuzzles::from(["".to_string()])),
+    ]))
+});
 
 /// without `name`, the app won't run
 fn ensure_env_var(key: &str) -> String {
